src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -17,279 +18,149 @@
-  <form method="GET" class="px3 mb4"
-    action="/samples_templates/hotel/check-available"
-    action-xhr="/samples_templates/hotel/check-available"
-    target="_top">
-    <fieldset id="stay-input" class="border-none p0 m0">
-    <div class="ampstart-input relative m0 p0 mb3">
-      <label for="checkin">Check-in:</label>
-      <input name="checkin" id="checkin" type="date" class="block border-none p0 m0" value="2020-12-30">
-    </div>
-    <div class="ampstart-input relative m0 p0 mb3">
-      <label for="checkout">Check-out:</label>
-      <input name="checkout" type="date" class="block border-none p0 m0" value="2021-01-06">
-    </div>
-    <div class="ampstart-input relative m0 p0 mb3 ">
-      <label for="guests">Guests:</label>
-      <select name="guests" class="block border-none p0 m0">
-        <option value="1">1</option>
-        <option value="2" selected>2</option>
-        <option value="3">3</option>
-      </select>
-    </div>
-    </fieldset>
-    <input type="submit" value="Check availability" class="ampstart-btn caps">
-    <div submit-success>
-      <template type="amp-mustache">
-        Yay!
-      </template>
-    </div>
-    <div submit-error>
-      <template type="amp-mustache">
-        No rooms available.
-      </template>
-    </div>
-  </form>
-</div>
-<!-- ## Available rooms -->
+<script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
 <!--
-  Another `amp-form` around the table allows for booking buttons.
+  We use `amp-date-picker` to implement a filter based on date selection.
 -->
-<div>
-    <h6>Rooms</h6>
-  <form method="POST" action-xhr="/samples_templates/hotel/book" target="_top">
-    <table>
-      <thead>
-        <tr>
-          <th>Type</th>
-          <th>Per night</th>
-          <th>Total</th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <td>Standard Room</td>
-          <td>$100 ave. per night</td>
-          <td>$700 total</td>
-          <td><input type="submit" value="Book it" class="ampstart-btn caps"></td>
-        </tr>
-        <tr>
-          <td>Junior Penthouse</td>
-          <td>$500 ave. per night</td>
-          <td>$3500 total</td>
-          <td><input type="submit" value="Book it" class="ampstart-btn caps"></td>
-        </tr>
-        <tr>
-          <td>Terrace Penthouse</td>
-          <td>$1000 ave. per night</td>
-          <td>$7000 total</td>
-          <td><input type="submit" value="Book it" class="ampstart-btn caps"></td>
-        </tr>
-      </tbody>
-    </table>
-  </form>
-</div>
-<!-- ## Embedding a map -->
+<script async custom-element="amp-date-picker" src="https://cdn.ampproject.org/v0/amp-date-picker-0.1.js"></script>
 <!--
-  A standard `amp-iframe` is used to embed a Google Map. Learn more about other iframe embeds [here](/components/amp-iframe).
+  We use `amp-lightbox-gallery` to implement a gallery of images.
```

Comment:
Can you improve: 'to implement a gallery of images.' - it doesn't really describe what and why we're using it.

Reply:
done